src/Approximations/box_approximations.jl

```diff
@@ -56,11 +56,25 @@ end
 box_approximation_symmetric(∅::EmptySet) = ∅
 
 """
-    symmetric_interval_hull
+    symmetric_interval_hull(X::LazySet)
 
-Alias for `box_approximation_symmetric`.
+Return a set that tightly overapproximates `X` with a hyperrectangular set symmetric
+respect to the origin.
 """
-symmetric_interval_hull = box_approximation_symmetric
+function symmetric_interval_hull(X::LazySet)
+    return box_approximation_symmetric(X)
+end
```

Comment:
Here `symmetric_interval_hull` was defined as an alias of `box_approximation_symmetric`. Hence you need to override the definition of `box_approximation_symmetric` instead. Otherwise that function will not make use of the new method.

Reply:
the point in this PR was to specialize `symmetric_interval_hull` for intervals. i didnt change `box_approximation_symmetric` in purpose: it stil remains a *box* (rectangle).

Reply:
i should probably add a test with the new behavior..

Reply:
I do not think that this is useful. They have just been aliases before and we have never cared for a difference.

Reply:
:man_shrugging:  do you care if i now care for a difference?

Reply:
Yes, this is a semantic change and I disagree.

Reply:
Fair enough, this change is untelated to the purpose of this pr anyways